### PR TITLE
Add test file with sorting columns

### PR DESCRIPTION
**Pull request by mapleFU**
Generate script: (with pyarrow 16.1.0 )

```python
>>> import pyarrow as pa
>>> import pyarrow.parquet as pq
>>> sorting_columns = (pq.SortingColumn(column_index=0, descending=True, nulls_first=True), pq.SortingColumn(column_index=1, descending=False))
>>> table = pa.table({'a': [None, 2, 1], 'b': ['a', 'b', 'c']})
>>> with pq.ParquetWriter('sorting_columns.parquet', table.schema, sorting_columns=sorting_columns) as writer:
...     for i in range(2):
...             writer.write_table(table)
```

File Metadata:

```json
{
  "Version": "2.6",
  "CreatedBy": "parquet-cpp-arrow version 16.1.0",
  "TotalRows": "6",
  "NumberOfRowGroups": "2",
  "NumberOfRealColumns": "2",
  "NumberOfColumns": "2",
  "Columns": [
     { "Id": "0", "Name": "a", "PhysicalType": "INT64", "ConvertedType": "NONE", "LogicalType": {"Type": "None"} },
     { "Id": "1", "Name": "b", "PhysicalType": "BYTE_ARRAY", "ConvertedType": "UTF8", "LogicalType": {"Type": "String"} }
  ],
  "RowGroups": [
     {
       "Id": "0",  "TotalBytes": "166",  "TotalCompressedBytes": "174",  "SortColumns": [{"column_idx":0, "descending":1, "nulls_first": 1}, {"column_idx":1, "descending":0, "nulls_first": 0}],  "Rows": "3",
       "ColumnChunks": [
          {"Id": "0", "Values": "3", "StatsSet": "True", "Stats": {"NumNulls": "1", "Max": "2", "Min": "1" },
           "Compression": "SNAPPY", "Encodings": "PLAIN(DICT_PAGE) RLE_DICTIONARY", "UncompressedSize": "100", "CompressedSize": "104" },
          {"Id": "1", "Values": "3", "StatsSet": "True", "Stats": {"NumNulls": "0", "Max": "c", "Min": "a" },
           "Compression": "SNAPPY", "Encodings": "PLAIN(DICT_PAGE) RLE_DICTIONARY", "UncompressedSize": "66", "CompressedSize": "70" }
        ]
     },
     {
       "Id": "1",  "TotalBytes": "166",  "TotalCompressedBytes": "174",  "SortColumns": [{"column_idx":0, "descending":1, "nulls_first": 1}, {"column_idx":1, "descending":0, "nulls_first": 0}],  "Rows": "3",
       "ColumnChunks": [
          {"Id": "0", "Values": "3", "StatsSet": "True", "Stats": {"NumNulls": "1", "Max": "2", "Min": "1" },
           "Compression": "SNAPPY", "Encodings": "PLAIN(DICT_PAGE) RLE_DICTIONARY", "UncompressedSize": "100", "CompressedSize": "104" },
          {"Id": "1", "Values": "3", "StatsSet": "True", "Stats": {"NumNulls": "0", "Max": "c", "Min": "a" },
           "Compression": "SNAPPY", "Encodings": "PLAIN(DICT_PAGE) RLE_DICTIONARY", "UncompressedSize": "66", "CompressedSize": "70" }
        ]
     }
  ]
}
```

### CODE REVIEW

**Comment by mapleFU** [comment]
cc @wgtmac @pitrou

**Comment by wgtmac** [comment]
If might be useful to have multiple row groups.

**Comment by mapleFU** [comment]
> If might be useful to have multiple row groups.

Currently, we're not able to create file with different sort-props in multiple row-groups. Should I first implement that or just same sort-properties in two row-groups?

**Comment by wgtmac** [comment]
I think it is fine to use the same properties across all row groups.

**Comment by mapleFU** [comment]
@wgtmac @pitrou 

I've change to use 2 row-groups with same data. Pr description is changed for this

File: data/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 | byte_stream_split.zstd.parquet | Standard normals with `BYTE_STREAM_SPLIT` encoding. See [note](#byte-stream-split) below |
 | incorrect_map_schema.parquet | Contains a Map schema without explicitly required keys, produced by Presto. See [note](#incorrect-map-schema) |
 | column_chunk_key_value_metadata.parquet | two INT32 columns, one with column chunk key-value metadata {"foo": "bar", "thisiskeywithoutvalue": null} note that the second key "thisiskeywithoutvalue", does not have a value, but the value can be mapped to an empty string "" when read depending on the client |
+| sorting_columns.parquet | INT64 and BYTE_ARRAY columns with first column with nulls_first and descending, second column with not nulls first and ascending. This file contains two row-groups with same data and sorting_columns. |
```
Comment (wgtmac) [inline data/README.md:55]:
```suggestion
| sorting_columns.parquet | INT64 and BYTE_ARRAY columns with first column with nulls first and descending, second column with nulls last and ascending. This file contains two row groups with same data and sorting columns. |
```

**Review by wgtmac — APPROVED** [review]
LGTM with a nit comment

**Comment by mapleFU** [comment]
I would merge this first. Feel free to edit or overwrite this if has any issue on this file or description.

**Pull request merged**